MapboxCoreNavigation/MMEEventsManager.swift

```diff
@@ -82,6 +90,27 @@ struct EventDetails {
         batteryPluggedIn = UIDevice.current.batteryState == .charging || UIDevice.current.batteryState == .full
         batteryLevel = UIDevice.current.batteryLevel >= 0 ? UIDevice.current.batteryLevel * 100 : -1
         applicationState = UIApplication.shared.applicationState
+        if let manager = routeController.locationManager {
+            locationEngine = type(of: manager)
+        }
+        
+        var totalTimeInPortrait = session.timeSpentInPortrait
+        var totalTimeInLandscape = session.timeSpentInLandscape
+        if UIDevice.current.orientation == .portrait {
```

Comment:
Checking back in the docs, we should use `isPortrait` and `isLandscape` everywhere instead of trying to parse these values ourselves: https://developer.apple.com/documentation/uikit/uideviceorientation